src/nodes.coffee

```diff
@@ -1366,6 +1366,11 @@ exports.Assign = class Assign extends Base
   isStatement: (o) ->
     o?.level is LEVEL_TOP and @context? and (@moduleDeclaration or "?" in @context)
 
+  checkAssignability: (o, varBase) ->
+    if Object::hasOwnProperty.call(o.scope.positions, varBase.value) and
+    o.scope.variables[o.scope.positions[varBase.value]].type is 'import'
```

Comment:
I think this looks really confusing when the line isn’t indented. I’d suggest adding three spaces at the beginning of the line.